ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,465 @@
+    @objc open var customWidth: CGFloat {
+        didSet {
+            if customWidth != oldValue {
+                setupLayoutConstraints()
+            }
+        }
+    }
+
+    /// The style of the card; vertical or horizontal. Style determines the layout of the content
+    private var style: CardStyle = .horizontal
+
+    /// The view of the icon. Appears on top of the text in vertical cards and besides the text in horizontal cards
+    private var iconView: UIImageView
+
+    /// The label for the Card's title
+    private let primaryLabel: Label = {
+        let primaryLabel = Label()
+        primaryLabel.translatesAutoresizingMaskIntoConstraints = false
+        primaryLabel.adjustsFontForContentSizeCategory = true
+        primaryLabel.font = TextStyle.subhead.font
+        primaryLabel.textColor = Colors.textPrimary
+        primaryLabel.textAlignment = .natural
+        return primaryLabel
+    }()
+
+    /// The label for the Card's subtitle
+    private let secondaryLabel: Label = {
+        let secondaryLabel = Label()
+        secondaryLabel.translatesAutoresizingMaskIntoConstraints = false
+        secondaryLabel.adjustsFontForContentSizeCategory = true
+        secondaryLabel.font = TextStyle.footnote.font
+        secondaryLabel.textColor = Colors.textSecondary
+        secondaryLabel.textAlignment = .natural
+        return secondaryLabel
+    }()
+
+    /// The base height of a vertical Card. It includes the top padding, the content spacing, the icon's height and the bottom padding
+    private var verticalBaseHeight: CGFloat
+
+    /// The base height of a horizontal Card. It includes the top and bottom paddings
+    private var horizontalBaseHeight: CGFloat
+
+    /// The height constraint of the Card. It's updated when the preferred size category changes
+    private var heightConstraint: NSLayoutConstraint
+
+    /**
+     Initializes `CardView`
+     
+     - Parameter style: The style of the card
+     - Parameter title: The title of the card
+     - Parameter subtitle: The subtitle of the card - optional
+     - Parameter icon: The icon of the card
+     - Parameter colorStyle: The Card's color style; appColor, neutral, or custom
+     **/
+    @objc public init(style: CardStyle,
+                      title: String,
+                      subtitle: String? = nil,
+                      icon: UIImage,
+                      colorStyle: CardColorStyle) {
+        self.primaryText = title
+        self.style = style
+        self.secondaryText = subtitle
+        self.icon = icon
+        self.colorStyle = colorStyle
+        customWidth = style.width
+        iconView = UIImageView(image: icon)
+
+        // Initialize the base heights
+        verticalBaseHeight = (Constants.verticalPaddingTop + Constants.iconHeight + Constants.verticalContentSpacing + Constants.verticalPaddingBottom)
+        horizontalBaseHeight = (2 * Constants.horizontalPadding)
+        heightConstraint = NSLayoutConstraint.init()
```

Comment:
This should be a lazy var. NSLayoutConstraint.init() is wasted since heightConstraint is reassigned later.
In the lazy var, initialize the heightAnchor constraint. Then, later, simply modify the constant and activate/deactivate the constraint when needed.